<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManageMainDialog</class>
 <widget class="QDialog" name="ManageMainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>435</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>435</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Main Configuration Manager</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMenuBar::item {background:#333333; color:#e0e0e0;}
QMenuBar::item:selected {background:#505050;}
QMenuBar::item:!enabled{background:#333333;color:#808080;}
QMenu::item:selected {background:#505050;}
QMenu::item:!enabled {background:#333333;color:#808080;}
QWidget {background-color:#303030; color:white;border:0px;}
QComboBox {background-color:#4A4A4A;}
QComboBox:hover{background-color:#405080;}
QComboBox:!enabled{background-color:#202020;color:#808080;}
QPushButton{background-color:#444;border:1px solid #888;width:80px; height:20px;border-radius:5px;}
QPushButton:!enabled{background-color:#202020;color:#808080}
QPushButton:hover{background-color:#405080;}
QPushButton:pressed {background:#505020;border:1px inset #222;}
QCheckBox:!enabled{color:#808080;}
QCheckBox::indicator:!enabled{background-color:#808080;color:#808080;}
QGroupBox{border:1px solid #407090; color:#407090;}
QSpinBox:!enabled {color:#777;}
QDoubleSpinBox:!enabled {color:#777;}
QGroupBox {background-color: rgba(60, 60, 60, 100);}
QLabel {background-color: rgba(0, 0, 0, 0);}
QLineEdit {border:1px solid #888;}</string>
  </property>
  <widget class="QPushButton" name="renamePushButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>1</horstretch>
     <verstretch>1</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Rename (A) to &gt;&gt;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="confAComboBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>1</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Configuration (A)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="confBComboBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>100</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>4</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>100</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="copyPushButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>5</horstretch>
     <verstretch>1</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Copy (A) to (B) &gt;&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="renameLineEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>160</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>1</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QPushButton" name="deletePushButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>1</horstretch>
     <verstretch>1</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Remove configuration (A)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closePushButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>260</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>1</horstretch>
     <verstretch>1</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="swapPushButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>1</horstretch>
     <verstretch>1</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Swap (A) with (B) &gt;&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>80</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>1</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Conf (B)</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
